designer/server/src/test/scala/pl/touk/nussknacker/ui/process/deployment/DeploymentServiceSpec.scala

```diff
@@ -109,10 +116,69 @@ class DeploymentServiceSpec
       processValidatorByProcessingType,
       TestFactory.scenarioResolverByProcessingType,
       listener,
-      scenarioStateTimeout = scenarioStateTimeout
+      scenarioStateTimeout = scenarioStateTimeout,
+      deploymentCommentSettings
     )
   }
 
+  // TODO: temporary step - we would like to extract the validation and the comment validation tests to external validators
+
+  private val nonEmptyCommentSettings = Some(createNonEmptyDeploymentCommentSettings())
```

Comment:
Lets make it more explicit, to be aware what does "nonEmpty" mean and what happens in tests:
```
private val nonEmptyCommentSettings = Some(DeploymentCommentSettings.unsafe(".+", Some("example")))
```